templates/nextflow-tower-config-bucket.yaml

```diff
@@ -0,0 +1,61 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: An S3 bucket readable by the account only
+
+Parameters:
+
+  BucketName:
+    Type: String
+    Description: Name of the created bucket.
+
+  AdminArns:
+    Type: CommaDelimitedList
+    Description: IAM ARNs that may administer and write to teh bucket
+    ConstraintDescription: >
+      List of ARNs (i.e. ['arn:aws:iam::011223344556:user/jsmith',
+      'arn:aws:iam::544332211006:user/rjones']
+
+Resources:
+
+  Bucket:
+    Type: AWS::S3::Bucket
+    DeletionPolicy: Delete
+    Properties:
+      BucketName: !Ref BucketName
+      AccessControl: Private
+      OwnershipControls:
+        Rules:
+          - ObjectOwnership: BucketOwnerPreferred
+
+  BucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref Bucket
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: FullAccess
+            Effect: Allow
+            Principal:
+              AWS: !Ref AdminArns
+            Action: 's3:*'
+            Resource:
+              - !Sub 'arn:aws:s3:::${Bucket}'
+              - !Sub 'arn:aws:s3:::${Bucket}/*'
```

Comment:
your passing in SSO administrators and ci service account for AdminArns. those users/roles already have full bucket access so I'm not sure why you need to give them access again?

Reply:
Good point -- removed in eef71a1